packages/application-generic/src/usecases/get-feature-flag/get-is-in-memory-cluster-mode-enabled.use-case.ts

```diff
@@ -7,9 +7,9 @@ import { GetSystemCriticalFlag } from './get-system-critical-flag.use-case';
 @Injectable()
 export class GetIsInMemoryClusterModeEnabled extends GetSystemCriticalFlag {
   execute(): boolean {
-    const value =
-      process.env.IS_IN_MEMORY_CLUSTER_MODE_ENABLED ??
-      process.env.IN_MEMORY_CLUSTER_MODE_ENABLED;
+    const value = process.env.IS_IN_MEMORY_CLUSTER_MODE_ENABLED
```

Comment:
Change needed as there was a failing test when `IS_IN_MEMORY_CLUSTER_MODE_ENABLED` was empty string. I added an extra test to confirm right behaviour.